hub/core/meta/meta.py

```diff
@@ -0,0 +1,97 @@
+from attr import Attribute
+from hub.util.exceptions import (
+    MetaAlreadyExistsError,
+    MetaDoesNotExistError,
+    MetaInvalidKey,
+    MetaInvalidRequiredMetaKey,
+)
+from hub.util.callbacks import (
+    CallbackDict,
+    convert_from_callback_objects,
+    convert_to_callback_objects,
+)
+import json
+from hub.core.storage.provider import StorageProvider
+
+import hub
+
+
+class Meta:
+    _initialized: bool = False
```

Comment:
this is necessary for `__setattr__` to work alongside the `storage` synchronization callback